### ISSUE TITLE
pypng is an unlisted dependency

### ISSUE THREAD

**Issue by rpdelaney** [mnooner256/pyqrcode#65]
Pretty much the subject. 

I installed this with `pip3 install --user pyqrcode` but got hit with module not found on line 1266 of builder.py: `import png`. The comments to that file state that pypng is required, so I installed that the same way I installed pyqrcode and now it works.

**Comment by agroszer** [comment]
Is this intentional? Just add pypng to `setup.py` `install_requires` please

**Comment by heuer** [comment]
We discussed it in the past since I wanted pypng as requirement but Michael said PNG is just one serialization format and wanted to keep it as optional dependency because all other formats work without further dependencies.

It's listed in the setup an extra:

>      extras_require = {
>        'PNG':  ["pypng>=0.0.13"],
> },

To cut a long story short: Yes, it is intentional. ;)

**Comment by agroszer** [comment]
ok, thx

**Comment by rpdelaney** [comment]
Optional is fine, but we should consider handling the `ImportError`, perhaps with a more user-friendly message.

**Comment by heuer** [comment]
I agree, anyway we cannot do anything about it; see <https://github.com/mnooner256/pyqrcode/issues/52>

And without solving <https://github.com/pypa/warehouse/issues/4121> I am not in the mood to invest any more energy into this project.
